example/lib/set_or_delete_location.dart

```diff
@@ -20,36 +20,35 @@ class SetOrDeleteLocationDialog extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return AlertDialog(
-      title: const Text('Enter location data'),
+      title: const Center(
+        child: Text('Enter location data'),
+      ),
       content: Column(
         mainAxisAlignment: MainAxisAlignment.center,
         mainAxisSize: MainAxisSize.min,
         children: [
           ElevatedButton(
-            onPressed: () {
-              showDialog<void>(
-                context: context,
-                builder: (context) => SetLocationDialog(
-                  id: id,
-                  name: name,
-                  geoFirePoint: geoFirePoint,
-                ),
-              );
-            },
+            onPressed: () => showDialog<void>(
+              context: context,
+              builder: (_) => SetLocationDialog(
+                id: id,
+                name: name,
+                geoFirePoint: geoFirePoint,
+              ),
+            ),
             child: const Text('set location'),
           ),
+          // child: const Text('set location'),
```

Comment:
Please remove this line!

Reply:
I appreciate it. 

[0e9c9f7](https://github.com/KosukeSaigusa/geoflutterfire_plus/pull/25/commits/0e9c9f7b0e97e5bfa5dda48f3d42e9d0956ce435)